src/main/java/uk/gov/hmcts/reform/sscs/service/AirLookupService.java

```diff
@@ -63,96 +56,77 @@ public void init() {
-                            lookupRegionalCentreByPostCode.put(postcodeCell.getRichStringCellValue().getString().toLowerCase(), adminGroupCell.getRichStringCellValue().getString());
-                        }
-
-                        if (lookupIdColumn != null && lookupIdColumn.getCellTypeEnum() == CellType.NUMERIC && String.valueOf(lookupIdColumn.getNumericCellValue()).equals("1.0")
-                                && postcodeCell != null && venueCell != null
-                                && postcodeCell.getCellTypeEnum() == CellType.STRING && venueCell.getCellTypeEnum() == CellType.STRING) {
-                            // Work out whether a string value has PIP in it and extract venue name
-                            // e.g. Northampton - 03 - PIP/DLA
-                            if (hasPip(venueCell.getRichStringCellValue().getString())) {
-                                String venueName = venueCell.getRichStringCellValue().getString();
-
-                                String venueNameSplitChar = "-";
-                                if (venueName.indexOf(venueNameSplitChar) != -1) {
-                                    lookupAirVenueNameByPostCode.put(postcodeCell.getRichStringCellValue().getString().toLowerCase().trim(),
-                                            venueName.substring(0, venueName.indexOf(venueNameSplitChar)).trim());
-                                } else {
-                                    String message = "Unknown venue name type" + venueName;
-                                    AirLookupServiceException ex = new AirLookupServiceException(new Exception(message));
-                                    LOG.error(message, ex);
-                                }
-                            } else {
-                                //Other benefit types to come
-                            }
-                        }
+    private void parseAirLookupData(Workbook wb)  {
+
+        for (Sheet sheet: wb) {
+            if (sheet.getSheetName().equals("All")) {
+                for (Row row : sheet) {
+                    if (row.getRowNum() == 0 || row.getRowNum() == 1) {
+                        continue;
+                    }
+                    Cell postcodeCell = row.getCell(POSTCODE_COLUMN);
+                    Cell adminGroupCell = row.getCell(REGIONAL_CENTRE_COLUMN);
+                    Cell esaCell = row.getCell(ESA_COLUMN);
+                    Cell pipCell = row.getCell(PIP_COLUMN);
+
+                    if (postcodeCell != null && adminGroupCell != null
+                            && postcodeCell.getCellTypeEnum() == CellType.STRING && adminGroupCell.getCellTypeEnum() == CellType.STRING) {
+                        lookupRegionalCentreByPostCode.put(postcodeCell.getRichStringCellValue().getString().toLowerCase().trim(), adminGroupCell.getRichStringCellValue().getString());
```

Comment:
It may be good to remove all spaces rather than trim, but if that's how it was working before it's also best not to throw more stones on the broken window.